### DISCUSSION TITLE
Request to merge `v-scope` into standard Vue 2/3

### DISCUSSION THREAD

**Discussion by caikan** [vuejs/rfcs#379] [General]
I like `v-scope` in petite-vue. It helps me write less boilerplate code for declaring local variables. Before that, I tried to achieve a similar effect with scoped slot or `v-for`. If this feature is built into the standard Vue, it will bring a lot of convenience.

## Demo with scoped slot
https://codesandbox.io/s/use-state-228br?file=/src/App.vue

## Demo with `v-if`
```vue
<div v-for="(obj, i) in [data.some.deep.prop]" :key="i">
  <div>{{obj.foo}}</div>
  <div>{{obj.bar}}</div>
  <div>{{obj.baz}}</div>
</div>
```

**Comment by posva** [comment]
Duplicate of https://github.com/vuejs/rfcs/discussions/505
